docs/02-app/02-api-reference/05-next-config-js/crossOrigin.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: crossOrigin
+description: Use the `crossOrigin` option to add a crossOrigin tag on the `script` tags generated by `next/script`. 
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+Use the `crossOrigin` option to add a [`crossOrigin` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin) in all `<script>` tags generated by the <AppOnly>[`next/script`](/docs/app/building-your-application/optimizing/scripts) component</AppOnly> <PagesOnly>[`next/script`](/docs/pages/building-your-application/optimizing/scripts) component and [`next/head`](/docs/pages/api-reference/components/head)components</PagesOnly>, and define how cross-origin requests should be handled.
```

Comment:
The option adds `crossorigin` to all Next.js internal script tags, i.e. the runtime and other bundles being loaded, it doesn't add it to `next/image` and `next/head` automatically.

Reply:
Sorry, the comment in the [source code](https://github.com/vercel/next.js/blob/b5155e166b44aa1a6730826d2f136a8a338c36ec/packages/next/src/server/config-shared.ts#L674) also says:

> Add `"crossorigin"` attribute to generated `<script>` elements generated by `<Head />` or `<NextScript />` components

Should we update the comment?